.github/ISSUE_TEMPLATE/ad-hoc-meeting.yaml

```diff
@@ -0,0 +1,53 @@
+name: 'Create an Ad-Hoc Meeting'
+description: 'This Issue template is used for creating a new Ad-Hoc Meeting for the Cross Project Council'
+title: "[Ad hoc meeting] "
+assignees:
+  - bensternthal
+labels:
+  - cross-project-council-agenda
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Agenda for the Meeting
+  - type: input
+    id: facilitator
+    attributes:
+      label: Facilitator
+      description: Who is the facilitator of this meeting?
+    validations:
+      required: true
+  - type: input
+    id: related-issues
+    attributes:
+      label: Related Issue(s)
+      description: What are the related Issues to this Meeting?
+    validations:
+      required: false
+  - type: input
+    id: meeting-date
+    attributes:
+      label: Meeting Date
+      description: 'When should the meeting happen? Use a MM/DD/YYY hh:mm TZ format (e.g.: 01/01/2001 00:00 UTC-3)'
```

Comment:
Can we please do ISO dates to avoid confusion for non-US folks.

Reply:
Definitely! But which ISO format? ISO-8601? `yyyy-MM-dd HH:mm SSZ`?